frontend/src/api/accountApi.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+import { Constants } from "../config/constants";
+
+const api = axios.create({
+  baseURL: `${Constants.idpUrl}/api`
+});
+
+const AccountApi = {
+  deleteAccount: (user, password) =>
+    api.post("/DeleteAccount", { Username: user, Password: password })
```

Comment:
Why do we pass the user and not taking it from the session ?

Reply:
good point! Had a chat with @RaduCStefanescu and my takeaway was we need to ask for them to type the email and password. It might be that I got it wrong and we only need to ask for the password. @RaduCStefanescu can you clarify if there is a requirement to ask the user to put in the email?

Reply:
Based on [this comment](https://github.com/code4romania/stam-acasa/pull/442#discussion_r446622573) I'm guessing he's on-board with the idea 😁

Reply:
cool! I will update the PR accordingly